Ovirt - orphaned cleanup null pointer

I get this error

2024/05/03 05:41:10 INFO    (main): Cleaning up orphaned objects...
panic: assignment to entry in nil map

goroutine 19 [running]:
github.com/bl4ko/netbox-ssot/internal/netbox/inventory.(*NetboxInventory).AddVM(0xc0000de1e0, {0xf3b8b8, 0xc0008be120}, 0xc0013d0540)
        C:/BIT/New folder/netbox-ssot/internal/netbox/inventory/add_items.go:761 +0x6dd
github.com/bl4ko/netbox-ssot/internal/source/ovirt.(*OVirtSource).syncVms(0xc00008c820, 0xc0000de1e0)
        C:/BIT/New folder/netbox-ssot/internal/source/ovirt/ovirt_sync.go:638 +0xe5
github.com/bl4ko/netbox-ssot/internal/source/ovirt.(*OVirtSource).Sync(0xc00008c820, 0xc0000de1e0)
        C:/BIT/New folder/netbox-ssot/internal/source/ovirt/ovirt.go:107 +0x2fb
main.main.func1({0xf3b8b8, 0xc0008be120}, {0xf39b48, 0xc00008c820})
        C:/BIT/New folder/netbox-ssot/cmd/netbox-ssot/main.go:98 +0x2f1
created by main.main in goroutine 1
        C:/BIT/New folder/netbox-ssot/cmd/netbox-ssot/main.go:78 +0x67e

--- a/internal/netbox/inventory/add_items.go
+++ b/internal/netbox/inventory/add_items.go
@@ -758,6 +758,9 @@ func (nbi *NetboxInventory) AddVM(ctx context.Context, newVM *objects.VM) (*obje
 		if err != nil {
 			return nil, err
 		}
+		if nbi.VMsIndexByNameAndClusterID[newVM.Name] == nil {
+			nbi.VMsIndexByNameAndClusterID[newVM.Name] = make(map[int]*objects.VM)
+		}
 		nbi.VMsIndexByNameAndClusterID[newVM.Name][newVMClusterID] = newVM
 		return newVM, nil
 	}

